src/code/z_vr_box.c

```diff
@@ -1,13 +1,677 @@
 #include "global.h"
+#include "vt.h"
 
+typedef struct {
+    /* 0x000 */ s32 unk_0;
+    /* 0x004 */ s32 unk_4;
+    /* 0x008 */ s32 unk_8;
+    /* 0x00C */ s32 unk_C;
+    /* 0x010 */ s32 unk_10;
+} Struct_8012AF0C; // size = 0x14
+
+extern Struct_8012AF0C D_8012AF0C[6];
+extern Struct_8012AF0C D_8012AEBC[4];
+
+typedef struct {
+    /* 0x00 */ u16 unk_0; // start
+    /* 0x02 */ u16 unk_2; // end
+    /* 0x04 */ u8 unk_4;
+    /* 0x05 */ u8 unk_5; // img idx 1
+    /* 0x06 */ u8 unk_6; // img idx 2
+    /* 0x07 */ char unk_7[0x1];
+} Struct_8011FC1C; // size = 0x8
```

Comment:
these structs should be easy to document considering they have comments already, and it would be a great improvement

Reply:
this is a struct defined by kankyo which is already named in my progress over there. this is going to be replaced anyway, so it doesnt really matter